portal/server/controllers/utilisation-report-service/record-correction/record-correction-sent/index.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from 'express';
+import { getFormattedReportPeriodWithLongMonth } from '@ukef/dtfs2-common';
+import { asLoggedInUserSession, LoggedInUserSession } from '../../../../helpers/express-session';
+import { PRIMARY_NAV_KEY } from '../../../../constants';
+import { RecordCorrectionSentViewModel } from '../../../../types/view-models/record-correction/record-correction-confirmation';
+
+const renderRecordCorrectionSentPage = (res: Response, viewModel: RecordCorrectionSentViewModel) =>
```

Comment:
I have just realised this.  We should not be passing req or res to functions and we have not been anywhere else apart from this folder.  Can this be put in getRecordCorrectionSent.  Also can we have a tech debt ticket to remove this from other folders.  Can assign it to me

Reply:
This is such a small function but adds a lot of type safety to how we call the view model and ensures we pass in the correct shape of object to the template which is really helpful and has saved me a lot of time whilst working on this project. I disagree with removing this.

Reply:
This one I cannot budge on and is an antipattern compared to the rest of the code.
I have nearly done the work on the other files to remove this. And regarding structure, you can make a const of the type required and pass that to res.render

Reply:
Will change

Reply:
https://ukef-dtfs.atlassian.net/browse/FN-3742 jsut about to push to the PR